website/src/components/TweetQuote/index.tsx

```diff
@@ -37,10 +38,10 @@ export default function TweetQuote({
       <figcaption>
         <Link to={profileUrl} rel="nofollow">
           <div className="avatar">
-            <img
+            <Image
               alt={name}
               className={clsx('avatar__photo', styles.avatarImg)}
-              src={avatar}
+              img={avatar}
```

Comment:
Ideal image is not designed to work with external URLs. `<img>` is fine in this case